.travis.yml

```diff
@@ -1,107 +0,0 @@
-language: node_js
-
-env:
-  global:
-    - NPM_CONFIG_LOGLEVEL=warn
-    - CLOUDSDK_CORE_DISABLE_PROMPTS=1
-    - ANDROID_API_LEVEL=29
```

Comment:
Don't we still need the Android CI right now? Deleting this shouldn't be until we migrate

Reply:
Good catch. I deleted this while I was working just to reduce the load on travis. Will reinstate